### DISCUSSION TITLE
if you kill the defensebot, the game will crash

### DISCUSSION THREAD

**Discussion by MJJJJN11** [FakeFishGames/Barotrauma#14157] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The game crashed when I kill a defensebot.

### Reproduction steps

As the title saying, the game crashed everytime when I kill the defensebot ,besides that, it crashed occasionally when I kill other character.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've fixed the issue in our private repo now https://github.com/FakeFishGames/Barotrauma-development/commit/10743475238a5fc9999ac437f23baba72e69b0fb, and will hotfix it as soon as we can. Sorry for the inconvenience!

**Comment by Rokvach** [comment]
Tested, working correctly. Closing discussion.
